service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/RespondentController.java

```diff
@@ -68,35 +65,22 @@ public AboutToStartOrSubmitCallbackResponse handleAboutToSubmit(@RequestBody Cal
     private List<String> validate(CaseDetails caseDetails) {
         ImmutableList.Builder<String> errors = ImmutableList.builder();
 
-        Map<String, Object> respondentsData =
-            (Map<String, Object>) defaultIfNull(caseDetails.getData().get("respondents"), null);
-
         if (caseDetails.getData().containsKey("respondents1")) {
-
             List<Map<String, Object>> migratedRespondentObject =
                 (List<Map<String, Object>>) caseDetails.getData().get("respondents1");
 
-            List<MigratedRespondent> migratedRespondents = migratedRespondentObject.stream()
+            List<Respondents> respondents = migratedRespondentObject.stream()
```

Comment:
Do we still want to call them "migratedRespondent" - the class is changed after all.

Reply:
removed all reference to migration in controller